data/vibe/data/serialization.d

```diff
@@ -2028,6 +2028,7 @@ unittest {
 
 unittest { // issue 1991 - @system property getters/setters does not compile
 	static class A {
+        @safe:
```

Comment:
Spaces vs. tabs

Reply:
I think I fixed this, I copy-pasted a tab character from the next line because I'm bad at emacs :)  Worth you double checking though